openapi/components/requestBodies/PatchOrganization.yaml

```
@@ -1,9 +1,11 @@
 content:
   application/json:
     schema:
-      allOf: 
+      allOf:
         - $ref: ../schemas/Organizations/Organization.yaml
         - properties:
             website:
               readOnly: true
+            reportCurrency:
+              readOnly: true
 description: Create Organization Request.
```

```suggestion
description: Update Organization Request.
```
Not relevant to this PR though.